CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ different versioning scheme, following the Haskell community's
 * C# NuGet version: TBD
 * C# Comm NuGet version: TBD
 
+### `gbc` and Bond compiler library ###
+
+* C++ codegen now generates additional copy and move constructors with allocator
```

Comment:
Minor: consider "C++ codegen now generates copy and move constructors with an allocator argument when a custom allocator is used."

Reply:
Updated.